newIDE/app/src/UI/Alert/ConfirmDialog.js

```diff
@@ -82,7 +83,12 @@ function ConfirmDialog(props: Props) {
             onRequestClose={props.onDismiss}
             onApply={props.onConfirm}
           >
-            <MarkdownText translatableSource={props.message} isStandaloneText />
+            <Text>
```

Comment:
Why add this? The font is not the right one?

Reply:
indeed, the markdown text does not use the default font if not wrapped in a Text, this was done in other places of the code. Wondering if that's an issue with the markdown component or if that's actually good to have control over how to display it